src/app/core/services/portal.service.ts

```diff
@@ -296,37 +297,72 @@ deleteProFormaEntry(id: number): Observable<any> {
-}
+  getPlantInformation(id: number): Observable<IPlantInformation> {
+    return this._portalApi.getPlantInformation(id);
+  }
 
-updateProjectProfile(data: any): Observable<any> {
-  return this._portalApi.updateProjectProfile(data);
-}
+  updatePlantInformation(data: any): Observable<any> {
+    return this._portalApi.updatePlantInformation(data);
+  }
 
-exportProjectProfiles(): void {
-  this._portalApi.exportProjectProfiles().subscribe(response => {
-    const contentDisposition = response.headers.get('Content-Disposition');
-    const filename = contentDisposition 
-      ? contentDisposition.split(';')[1].trim().split('=')[1] 
-      : 'Library_Type_Options_' + this._datePipe.transform(new Date(), 'medium') + '.xlsx';
+  exportPlantInformations(): void {
+    this._portalApi.exportPlantInformations().subscribe(response => {
+      const contentDisposition = response.headers.get('Content-Disposition');
+      const filename = contentDisposition 
+        ? contentDisposition.split(';')[1].trim().split('=')[1] 
+        : 'Library_Type_Options_' + this._datePipe.transform(new Date(), 'medium') + '.xlsx';
 
-      const blob = new Blob([response.body], { type: 'application/actet-stream' });
+        const blob = new Blob([response.body], { type: 'application/actet-stream' });
+  
+        const link = document.createElement('a');
+        link.href = window.URL.createObjectURL(blob);
+        link.download = filename;
+        link.click();
+      });
+    }
 
-      const link = document.createElement('a');
-      link.href = window.URL.createObjectURL(blob);
-      link.download = filename;
-      link.click();
-    });
+  createPlantInformation(data: any): Observable<any> {
+    return this._portalApi.createPlantInformation(data);
   }
 
-createProjectProfile(data: any): Observable<any> {
-  return this._portalApi.createProjectProfile(data);
-}
+  getProjectProfiles(): Observable<IProjectProfile[]> {
+    return this._portalApi.getProjectProfiles();
+  }
+  
+  getProjectProfile(id: number): Observable<IProjectProfile> {
+    return this._portalApi.getProjectProfile(id);
+  }
+  
+  createProjectProfile(data: any): Observable<any> {
+    return this._portalApi.createProjectProfile(data);
+  }
+  
+  updateProjectProfile(data: any): Observable<any> {
+    return this._portalApi.updateProjectProfile(data);
+  }
+  
+  exportProjectProfiles(): void {
+    this._portalApi.exportProjectProfiles().subscribe(response => {
+      const contentDisposition = response.headers.get('Content-Disposition');
+      const filename = contentDisposition 
+        ? contentDisposition.split(';')[1].trim().split('=')[1] 
+        : 'Library_Type_Options_' + this._datePipe.transform(new Date(), 'medium') + '.xlsx';
+  
+        const blob = new Blob([response.body], { type: 'application/actet-stream' });
+  
+        const link = document.createElement('a');
+        link.href = window.URL.createObjectURL(blob);
```

Comment:
Why are these included in this PR? This is not related in Plant Information

Reply:
Kindly see update